### ISSUE TITLE
nsdiff doesn't detect change in SOA MNAME/RNAME

### ISSUE THREAD

**Issue by jpmens** [fanf2/nsdiff#4]
I think `nsdiff-1.73` doesn't detect a change of MNAME/RNAME in the SOA unless an additional change is performed on the zone. (Admittedly this is a very extreme edge case. )

Consider the following existing zone file _new.db_ which corresponds to the transferred zone:

```
$ head -1 new.db
jpm006.de. 3600 IN SOA bb. hostmaster.bb. 33   3600 1800 3600000 600

$ nsdiff -S serial -s 127.0.0.1 -u  jpm006.de new.db
nsdiff: loading zone jpm006.de. via AXFR from 127.0.0.1
zone jpm006.de/IN: loaded serial 16
OK
nsdiff: loading zone jpm006.de. from file new.db
new.db:8: using RFC1035 TTL semantics
zone jpm006.de/IN: loaded serial 33
OK
```

I now update MNAME and RNAME in the new zone file:

```
$ head -1 new.db
jpm006.de. 3600 IN SOA FANF. hostmaster.FANF. 33   3600 1800 3600000 600

$ nsdiff -S serial -s 127.0.0.1 -u  jpm006.de new.db
nsdiff: loading zone jpm006.de. via AXFR from 127.0.0.1
zone jpm006.de/IN: loaded serial 16
OK
nsdiff: loading zone jpm006.de. from file new.db
new.db:8: using RFC1035 TTL semantics
zone jpm006.de/IN: loaded serial 33
OK
```

In order to prove my point, I add a record to the new zone

```
$ echo "fff IN A 1.0.0.1" >> new.db
$ nsdiff -S serial -s 127.0.0.1 -u  jpm006.de new.db
nsdiff: loading zone jpm006.de. via AXFR from 127.0.0.1
zone jpm006.de/IN: loaded serial 16
OK
nsdiff: loading zone jpm006.de. from file new.db
new.db:8: using RFC1035 TTL semantics
zone jpm006.de/IN: loaded serial 33
OK
server 127.0.0.1
prereq yxrrset jpm006.de. IN SOA        bb. hostmaster.bb. 16 3600 1800 3600000 600
update add jpm006.de. 3600 IN SOA       FANF. hostmaster.FANF. 17 3600 1800 3600000 600
update add fff.jpm006.de. 340 IN A              1.0.0.1
send
answer
```

This should probably just be mentioned in the fine manual.

**Comment by fanf2** [comment]
Fixed in nsdiff-1.75

**Comment by jpmens** [comment]
Confirmed working. Thank you.
